### ISSUE TITLE
configure.ac problem detected in osx

### ISSUE THREAD

**Issue by psxdev** [psp2dev/psp2lib#1]
code:
        if test "$ac_compiler_gnu" = yes; then
    CFLAGS="$CFLAGS -Wall -mcpu=cortex-a9 -mfpu=neon-fp16"
    if test "$with_fpu" = yes; then
        CFLAGS="$CFLAGS -mfloat-abi=hard"
        psp2dir="$psp2dir/fpu"
    fi
        fi
It never goes inside "$ac_compiler_gnu"  if , 
so psp2dir never take good value when fpu is enabled

can be fixed without if code for ac_compiler_gnu
CFLAGS="$CFLAGS -Wall -mcpu=cortex-a9 -mfpu=neon-fp16"

```
if test "$with_fpu" = yes; then
    CFLAGS="$CFLAGS -mfloat-abi=hard"
    psp2dir="$psp2dir/fpu"
fi
```

**Comment by 173210** [comment]
It works fine on my PC. What compiler are you using?

**Comment by psxdev** [comment]
gnu arm-none-eabi for psp2 code
clang(gcc) for native osx code 

toolchain from my repo https://github.com/psxdev/psp2toolchain

**Comment by 173210** [comment]
It's strange. Anyway, we should remove $ac_compiler_gnu.
By the way, $with_fpu is not related to those flags, so I'll just remove the check.

**Comment by 173210** [comment]
Fixed with commit a881a2940c37ec6007da80f8433ddf26e3ed6007.
